lib/reply.js

```diff
@@ -164,7 +164,7 @@ Reply.prototype.send = function (payload) {
       if (hasContentType === false) {
         this[kReplyHeaders]['content-type'] = CONTENT_TYPE.OCTET
       }
-      const payloadToSend = Buffer.isBuffer(payload) ? payload : Buffer.from(payload.buffer)
+      const payloadToSend = payload instanceof Uint8Array ? payload : Buffer.from(payload.buffer)
```

Comment:
```suggestion
      const payloadToSend = Buffer.isBuffer(payload) ? payload : Buffer.from(payload.buffer, payload.byteOffset, payload.byteLength) 
```

I was referring to this form.

Reply:
Ah.. good catch. sorry about that.. that does change less things :)